static/lang/en.json

```diff
@@ -2800,6 +2800,7 @@
             "Label": "Legal Notice",
             "LandingPageText": "Use the tabs above to read the legal information"
         },
+        "Living": "Living",
```

Comment:
Consider putting this in re-en.json in SpecificRule.

Reply:
I was thinking maybe we consider an object for mode of living.